src/mixins/audit-wrapper.mixin.ts

```diff
@@ -0,0 +1,20 @@
+import {DefaultCrudRepository, Entity} from '@loopback/repository';
+import {AbstractConstructor, IAuditMixinOptions} from '../types';
+import {AuditRepositoryMixin} from './audit.mixin';
+
+export function auditRepositoryWrapper<
```

Comment:
Rename it to `ConditionalAuditRepositoryMixin` and please add some test cases for it.